Exceptions on exporting are logged as errors

**Describe the bug**

We currently log in the ExporterContainer all exceptions which happen during exporting as errors
https://github.com/zeebe-io/zeebe/blob/develop/broker/src/main/java/io/zeebe/broker/exporter/stream/ExporterContainer.java#L143

This causes a lot of errors in our Error reporting tool, since it can happen that elastic is not available. 
https://console.cloud.google.com/errors/CK7miMHiuIbORA?service=zeebe&time=P7D&project=camunda-cloud-240911&authuser=1

```
logging  exception: "io.zeebe.exporter.ElasticsearchExporterException: Failed to flush all items of the bulk
	at io.zeebe.exporter.ElasticsearchClient.flush(ElasticsearchClient.java:156) ~[zeebe-elasticsearch-exporter-0.25.0-alpha1.jar:0.25.0-alpha1]
	at io.zeebe.exporter.ElasticsearchExporter.flush(ElasticsearchExporter.java:116) ~[zeebe-elasticsearch-exporter-0.25.0-alpha1.jar:0.25.0-alpha1]
	at io.zeebe.exporter.ElasticsearchExporter.export(ElasticsearchExporter.java:84) ~[zeebe-elasticsearch-exporter-0.25.0-alpha1.jar:0.25.0-alpha1]
	at io.zeebe.broker.exporter.stream.ExporterDirector$RecordExporter.export(ExporterDirector.java:350) ~[zeebe-broker-0.25.0-alpha1.jar:0.25.0-alpha1]
	at io.zeebe.util.retry.BackOffRetryStrategy.run(BackOffRetryStrategy.java:51) ~[zeebe-util-0.25.0-alpha1.jar:0.25.0-alpha1]
	at io.zeebe.util.sched.ActorJob.invoke(ActorJob.java:76) [zeebe-util-0.25.0-alpha1.jar:0.25.0-alpha1]
	at io.zeebe.util.sched.ActorJob.execute(ActorJob.java:39) [zeebe-util-0.25.0-alpha1.jar:0.25.0-alpha1]
	at io.zeebe.util.sched.ActorTask.execute(ActorTask.java:118) [zeebe-util-0.25.0-alpha1.jar:0.25.0-alpha1]
	at io.zeebe.util.sched.ActorThread.executeCurrentTask(ActorThread.java:107) [zeebe-util-0.25.0-alpha1.jar:0.25.0-alpha1]
	at io.zeebe.util.sched.ActorThread.doWork(ActorThread.java:91) [zeebe-util-0.25.0-alpha1.jar:0.25.0-alpha1]
	at io.zeebe.util.sched.ActorThread.run(ActorThread.java:204) [zeebe-util-0.25.0-alpha1.jar:0.25.0-alpha1]
"   
  logger: "io.zeebe.broker.exporter.elasticsearch"   
  message: "Error on exporting record with key 2251799813691530" 
```

**To Reproduce**

Run our setup with elastic.

**Expected behavior**

Logged as warning.

**Log/Stacktrace**

<details><summary>Full Stacktrace</summary>
 <p>

```
io.zeebe.exporter.ElasticsearchExporterException: Failed to flush all items of the bulk
	at io.zeebe.exporter.ElasticsearchClient.flush(ElasticsearchClient.java:156) ~[zeebe-elasticsearch-exporter-0.25.0-alpha1.jar:0.25.0-alpha1]
	at io.zeebe.exporter.ElasticsearchExporter.flush(ElasticsearchExporter.java:116) ~[zeebe-elasticsearch-exporter-0.25.0-alpha1.jar:0.25.0-alpha1]
	at io.zeebe.exporter.ElasticsearchExporter.export(ElasticsearchExporter.java:84) ~[zeebe-elasticsearch-exporter-0.25.0-alpha1.jar:0.25.0-alpha1]
	at io.zeebe.broker.exporter.stream.ExporterDirector$RecordExporter.export(ExporterDirector.java:350) ~[zeebe-broker-0.25.0-alpha1.jar:0.25.0-alpha1]
	at io.zeebe.util.retry.BackOffRetryStrategy.run(BackOffRetryStrategy.java:51) ~[zeebe-util-0.25.0-alpha1.jar:0.25.0-alpha1]
	at io.zeebe.util.sched.ActorJob.invoke(ActorJob.java:76) [zeebe-util-0.25.0-alpha1.jar:0.25.0-alpha1]
	at io.zeebe.util.sched.ActorJob.execute(ActorJob.java:39) [zeebe-util-0.25.0-alpha1.jar:0.25.0-alpha1]
	at io.zeebe.util.sched.ActorTask.execute(ActorTask.java:118) [zeebe-util-0.25.0-alpha1.jar:0.25.0-alpha1]
	at io.zeebe.util.sched.ActorThread.executeCurrentTask(ActorThread.java:107) [zeebe-util-0.25.0-alpha1.jar:0.25.0-alpha1]
	at io.zeebe.util.sched.ActorThread.doWork(ActorThread.java:91) [zeebe-util-0.25.0-alpha1.jar:0.25.0-alpha1]
	at io.zeebe.util.sched.ActorThread.run(ActorThread.java:204) [zeebe-util-0.25.0-alpha1.jar:0.25.0-alpha1]
```

</p>
</details>


**Environment:**
 - OS: k8s
 - Zeebe Version: 0.25.0-alpha1
 - Configuration: elastic exporter

This will also happen when DNS resolution problems happen like:

```
io.zeebe.exporter.ElasticsearchExporterException: Failed to put index template
	at io.zeebe.exporter.ElasticsearchClient.putIndexTemplate(ElasticsearchClient.java:220) ~[zeebe-elasticsearch-exporter-0.23.6.jar:0.23.6]
	at io.zeebe.exporter.ElasticsearchClient.putIndexTemplate(ElasticsearchClient.java:209) ~[zeebe-elasticsearch-exporter-0.23.6.jar:0.23.6]
	at io.zeebe.exporter.ElasticsearchExporter.createRootIndexTemplate(ElasticsearchExporter.java:183) ~[zeebe-elasticsearch-exporter-0.23.6.jar:0.23.6]
	at io.zeebe.exporter.ElasticsearchExporter.createIndexTemplates(ElasticsearchExporter.java:136) ~[zeebe-elasticsearch-exporter-0.23.6.jar:0.23.6]
	at io.zeebe.exporter.ElasticsearchExporter.export(ElasticsearchExporter.java:95) ~[zeebe-elasticsearch-exporter-0.23.6.jar:0.23.6]
	at io.zeebe.broker.exporter.stream.ExporterDirector$RecordExporter.export(ExporterDirector.java:354) ~[zeebe-broker-0.23.6.jar:0.23.6]
	at io.zeebe.util.retry.BackOffRetryStrategy.run(BackOffRetryStrategy.java:51) ~[zeebe-util-0.23.6.jar:0.23.6]
	at io.zeebe.util.sched.ActorJob.invoke(ActorJob.java:76) [zeebe-util-0.23.6.jar:0.23.6]
	at io.zeebe.util.sched.ActorJob.execute(ActorJob.java:39) [zeebe-util-0.23.6.jar:0.23.6]
	at io.zeebe.util.sched.ActorTask.execute(ActorTask.java:115) [zeebe-util-0.23.6.jar:0.23.6]
	at io.zeebe.util.sched.ActorThread.executeCurrentTask(ActorThread.java:107) [zeebe-util-0.23.6.jar:0.23.6]
	at io.zeebe.util.sched.ActorThread.doWork(ActorThread.java:91) [zeebe-util-0.23.6.jar:0.23.6]
	at io.zeebe.util.sched.ActorThread.run(ActorThread.java:195) [zeebe-util-0.23.6.jar:0.23.6]
Caused by: java.io.IOException: elasticsearch
	at org.elasticsearch.client.RestClient$SyncResponseListener.get(RestClient.java:964) ~[elasticsearch-rest-client-6.8.9.jar:6.8.9]
	at org.elasticsearch.client.RestClient.performRequest(RestClient.java:233) ~[elasticsearch-rest-client-6.8.9.jar:6.8.9]
	at org.elasticsearch.client.RestHighLevelClient.internalPerformRequest(RestHighLevelClient.java:1764) ~[elasticsearch-rest-high-level-client-6.8.9.jar:6.8.9]
	at org.elasticsearch.client.RestHighLevelClient.performRequest(RestHighLevelClient.java:1734) ~[elasticsearch-rest-high-level-client-6.8.9.jar:6.8.9]
	at org.elasticsearch.client.RestHighLevelClient.performRequestAndParseEntity(RestHighLevelClient.java:1696) ~[elasticsearch-rest-high-level-client-6.8.9.jar:6.8.9]
	at org.elasticsearch.client.IndicesClient.putTemplate(IndicesClient.java:1504) ~[elasticsearch-rest-high-level-client-6.8.9.jar:6.8.9]
	at io.zeebe.exporter.ElasticsearchClient.putIndexTemplate(ElasticsearchClient.java:217) ~[zeebe-elasticsearch-exporter-0.23.6.jar:0.23.6]
	... 12 more
Caused by: java.net.UnknownHostException: elasticsearch
	at java.net.InetAddress$CachedAddresses.get(Unknown Source) ~[?:?]
	at java.net.InetAddress.getAllByName0(Unknown Source) ~[?:?]
	at java.net.InetAddress.getAllByName(Unknown Source) ~[?:?]
	at java.net.InetAddress.getAllByName(Unknown Source) ~[?:?]
	at org.apache.http.impl.conn.SystemDefaultDnsResolver.resolve(SystemDefaultDnsResolver.java:45) ~[httpclient-4.5.12.jar:4.5.12]
	at org.apache.http.impl.nio.conn.PoolingNHttpClientConnectionManager$InternalAddressResolver.resolveRemoteAddress(PoolingNHttpClientConnectionManager.java:664) ~[httpasyncclient-4.1.4.jar:4.1.4]
	at org.apache.http.impl.nio.conn.PoolingNHttpClientConnectionManager$InternalAddressResolver.resolveRemoteAddress(PoolingNHttpClientConnectionManager.java:635) ~[httpasyncclient-4.1.4.jar:4.1.4]
	at org.apache.http.nio.pool.AbstractNIOConnPool.processPendingRequest(AbstractNIOConnPool.java:474) ~[httpcore-nio-4.4.13.jar:4.4.13]
	at org.apache.http.nio.pool.AbstractNIOConnPool.lease(AbstractNIOConnPool.java:280) ~[httpcore-nio-4.4.13.jar:4.4.13]
	at org.apache.http.impl.nio.conn.PoolingNHttpClientConnectionManager.requestConnection(PoolingNHttpClientConnectionManager.java:295) ~[httpasyncclient-4.1.4.jar:4.1.4]
	at org.apache.http.impl.nio.client.AbstractClientExchangeHandler.requestConnection(AbstractClientExchangeHandler.java:377) ~[httpasyncclient-4.1.4.jar:4.1.4]
	at org.apache.http.impl.nio.client.DefaultClientExchangeHandlerImpl.start(DefaultClientExchangeHandlerImpl.java:129) ~[httpasyncclient-4.1.4.jar:4.1.4]
	at org.apache.http.impl.nio.client.InternalHttpAsyncClient.execute(InternalHttpAsyncClient.java:141) ~[httpasyncclient-4.1.4.jar:4.1.4]
	at org.elasticsearch.client.RestClient.performRequestAsync(RestClient.java:537) ~[elasticsearch-rest-client-6.8.9.jar:6.8.9]
	at org.elasticsearch.client.RestClient.performRequestAsyncNoCatch(RestClient.java:520) ~[elasticsearch-rest-client-6.8.9.jar:6.8.9]
	at org.elasticsearch.client.RestClient.performRequest(RestClient.java:232) ~[elasticsearch-rest-client-6.8.9.jar:6.8.9]
	at org.elasticsearch.client.RestHighLevelClient.internalPerformRequest(RestHighLevelClient.java:1764) ~[elasticsearch-rest-high-level-client-6.8.9.jar:6.8.9]
	at org.elasticsearch.client.RestHighLevelClient.performRequest(RestHighLevelClient.java:1734) ~[elasticsearch-rest-high-level-client-6.8.9.jar:6.8.9]
	at org.elasticsearch.client.RestHighLevelClient.performRequestAndParseEntity(RestHighLevelClient.java:1696) ~[elasticsearch-rest-high-level-client-6.8.9.jar:6.8.9]
	at org.elasticsearch.client.IndicesClient.putTemplate(IndicesClient.java:1504) ~[elasticsearch-rest-high-level-client-6.8.9.jar:6.8.9]
	at io.zeebe.exporter.ElasticsearchClient.putIndexTemplate(ElasticsearchClient.java:217) ~[zeebe-elasticsearch-exporter-0.23.6.jar:0.23.6]
	... 12 more
```

--- a/broker/src/main/java/io/zeebe/broker/exporter/stream/ExporterContainer.java
+++ b/broker/src/main/java/io/zeebe/broker/exporter/stream/ExporterContainer.java
@@ -140,7 +140,7 @@ final class ExporterContainer implements Controller {
       }
       return true;
     } catch (final Exception ex) {
-      context.getLogger().error("Error on exporting record with key {}", typedEvent.getKey(), ex);
+      context.getLogger().warn("Error on exporting record with key {}", typedEvent.getKey(), ex);
       return false;
     }
   }

